packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/simulationservice/SimulationServiceProxy.java

```diff
@@ -0,0 +1,34 @@
+package software.uncharted.terarium.hmiserver.proxies.simulationservice;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import software.uncharted.terarium.hmiserver.models.simulationservice.SimulationParams;
+
+
+@RegisterRestClient(configKey = "simulation-service")
+@Produces(MediaType.APPLICATION_JSON)
+public interface SimulationServiceProxy {
+	@POST
+	@Path("/calls/forecast")
+	@Consumes(MediaType.APPLICATION_JSON)
+	Response makeForecastRun(
+		SimulationParams simulationParams
+	);
+
+	@GET
+	@Path("/runs/{runId}/status")
+	@Consumes(MediaType.APPLICATION_JSON)
+	Response getRunStatus(
+		@PathParam("runId") String runId
+	);
+
+	@GET
+	@Path("/runs/{runId}/result")
+	@Consumes(MediaType.APPLICATION_JSON)
+	Response getRunResult(
+		@PathParam("runId") String runId
+	);
```

Comment:
`/status` doesn't return the data once it's ready? We have to call `/result` for it?

Reply:
I suspect separate endpoints should be fine.

Imagine that we have a listing of runs that we want to check (e.g. a summary table) and we just want to know if they are done or not, having a dedicated status end point accomplishes that without incurring the cost of sending the data.